### PR TITLE
598 create a landing page uzma

### LINKED ISSUE #598: Create a Landing Page
## **Changes:**

Must be an component named `LandingPage<YourName>`. Also make sure that your branch is renamed with the suffix `-yourname`.

Please create a folder with the following path `client/src/pages/Initial/LandingPage<YourName>`, inside the folder, make sure to add a .jsx file and a .scss file 👍 

The purpose of the landing page is to let F!rosh know that F!rosh week is coming, this page will appear when users navigate to orientation.skule.ca. Additionally, it will include some contact links/emails that F!rosh can begin to look into. The overall design should be enticing and inviting, making F!rosh excited for F!rosh week.

## **Visuals:**

Visuals are up to you, we want to give you all this task as a chance as an intro to React and getting used to Git/GitHub and making PRs.
Just make sure that the page is the screen size, and it should be responsive to multiple devices (laptop and phone).

## **Requirements:**

- [ ] Everyone will be making their own landing page -- take this as an opportunity to practice and freshen up your React skills! Make sure to add a suffix like `-yourname` on your branch name :)
- [ ] Follow the our branding colours 
- [ ] Include a message indicating that F!rosh Week is approaching
- [ ] Include the social media link for [F!rosh](https://www.instagram.com/froshweek/)  and the social media link for [leedurs/upper years](https://www.instagram.com/froshnomore/)

Note: There is no need to add any email/sign up things, and please implement the component as if you are using light mode.

## **External Factors:**

No external factors

## **Related Issues/PRs:**

No related Issues

## **Resources:**

Link any resources or people that might be helpful in the accomplishment of the issue.

### **Tutorials:**

- [Frontend Onboarding Package](https://docs.orientation.skule.ca/pages/Onboarding/Frontend.html)

### **Documentation:**

- [Example Landing Pages with Similar Vibes](https://www.google.com/search?q=under+construction+landing+page&tbm=isch&ved=2ahUKEwiwjpuPr4X_AhUhCVkFHbEhD0YQ2-cCegQIABAA&oq=under+con&gs_lcp=CgNpbWcQARgAMgcIABCKBRBDMgcIABCKBRBDMgUIABCABDIHCAAQigUQQzIFCAAQgAQyBQgAEIAEMgUIABCABDIFCAAQgAQyBQgAEIAEMgUIABCABDoECCMQJzoICAAQgAQQsQM6CggAEIoFELEDEENQmQ5Yhhxg3ypoAHAAeACAAVOIAeIFkgECMTCYAQCgAQGqAQtnd3Mtd2l6LWltZ8ABAQ&sclient=img&ei=ZIJpZLDMMaGS5NoPscO8sAQ&bih=818&biw=852&rlz=1C1RXQR_enCA1035CA1035)
- [Previous Landing Page Example](https://github.com/UofT-Frosh-Orientation/orientation-website/tree/dev/client/src/pages/Initial)

### PR DESCRIPTION
**Pull request by uzFer**
## **Contribution:**
Created super basic landing page with a logo, links, title, and subheading in a similar font as the previous homepage.

### **Issue:**
Closes #598 

### **Accomplishments:**
- Let users know the website is under construction/coming soon
- Social media links for F!rosh and Leedurs to navigate to

### **Visuals:** 
- Simple logo that was in the old navbar
- Instagram logo for Frosh week and Leedur accounts

### **Changes (05/27/2023):** 
- Fixed class names to be more specific (i.e. suffix -uzma)
- Updated scss file for the images with proper sizing
- Added links and Instagram logo for both accounts listed in issue
- Moved LandingPageUzma.jsx and LandingPageUzma.scss into pages folder

### CODE REVIEW

File: client/src/Initial/LandingPageUzma.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import './LandingPageUzma.scss';
+import MainFroshLogo from '../assets/logo/frosh-main-logo-with-bg.svg';
+
+const LandingPageUzma = () => {
+  return (
+    <div className="landing-page-header">
+      <div className="landing-page-header-text">
+        <img className="icon-logo" src={MainFroshLogo} alt="frosh logo"></img>
```
Comment (natapokie) [inline client/src/Initial/LandingPageUzma.jsx]: not entirely sure about the exact style that you're looking for, but a note for the future keep in mind that the class `icon-logo` is already defined, at least for React a class defined in another file can still be applied to other components.

so just make sure that your classnames are very unique! make sure to change all the instances of `landing-page` to `landing-page-uzma` or something similar to make sure that your classes don't affect other components

File: client/src/Initial/LandingPageUzma.scss [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  min-height: 500px;
+  background-image: linear-gradient(
+    to right,
+    var(--purple-transparent) 20%,
+    var(--purple-transparent-more)
+  );
+  display: flex;
+  align-items: center;
+  img {
+    height: max(10vh, 100px);
```
Comment (natapokie) [inline client/src/Initial/LandingPageUzma.scss]: react is a little funny (at least with what I've noticed) if you're trying to change the size of images...
if this line is intended to change the size of the image, you'll need to define the `height` _and_ `width`

File: client/src/App.jsx [hunk 3]
```diff
@@ -46,38 +47,5 @@ const TransitionRoutes = () => {
...
-      <CSSTransition key={location.key} classNames="page" timeout={300}>
-        <Routes location={location}>
-          {[
-            ...pages.main,
-            ...pages.hidden,
-            ...pages.special,
-            ...pages.scunt,
-            ...pages.scuntHidden,
-          ].map((page) => {
-            return (
```
Comment (natapokie) [inline client/src/App.jsx]: uncommit the deletions here
you're welcome to test the component by adding it directly in the App.jsx file, but make sure to remove it after testing -- we'll be setting up another component to display all the landing pages!

**Review by natapokie — CHANGES_REQUESTED** [review]
great work so far! please take a look at the comments, and resolve the suggestions :)

and a few more things... if you can actually move the files into `client/src/pages/Initial`, I mistyped the path in the original issue and make sure to include the social media link for F!rosh and the social media link for leedurs/upper years (links are included on the issue)

File: client/src/Initial/LandingPageUzma.jsx (hunk 1 above)
Comment (uzFer) [reply client/src/Initial/LandingPageUzma.jsx]: Just fixed that, that was me being lazy my bad 😅

File: client/src/pages/Initial/LandingPageUzma.jsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+        <a href="https://www.instagram.com/froshweek/" target="_blank" rel="noreferrer">
+          <img
+            src={InstagramLogo}
+            className="landing-page-uzma-instagram-logo"
+            alt="Instagram logo links to F!rosh Instagram page"
+          />
+        </a>
+        <p>F!rosh Week</p>
+        <a href="https://www.instagram.com/froshnomore/" target="_blank" rel="noreferrer">
```
Comment (natapokie) [inline client/src/pages/Initial/LandingPageUzma.jsx:23]: we have a class within the `App.scss` that removes the blue underline from the link tags called `no-link-style`.
image
you can use it like `className="no-link-style"` inside the `<a>` tags if you'd like.
if it's intention you can disregard the comment!

**Review by Tanushanth — APPROVED** [review]
Works fine! I tested it out and it looks good. Only thing is for some reason I had to change the export default LandingPageUzma to export {LandingPageUzma}, though I'm pretty sure that's fine.

**Comment by natapokie** [comment]
> Works fine! I tested it out and it looks good. Only thing is for some reason I had to change the export default LandingPageUzma to export {LandingPageUzma}, though I'm pretty sure that's fine.

Thanks for pointing that out Tanu! It seems to be just some React syntax, so just a note for the future, so typically, when you create your component and you're only exporting _**one component**_ you can use the `default` keyword, 
```
export default LandingPage = () => {//...}
```

otherwise, if you want to export multiple components, you would use the `const` keyword and then export them at the end of the file using the curly braces
```
const LandingPage1 = () => {//..}
const LandingPage2 = () => {//..}

export { LandingPage1, LandingPage2 }
```

Here's an [answer](https://stackoverflow.com/questions/33611812/export-const-vs-export-default-in-es6) from StackOverflow that explains it as well!

**Review by natapokie — APPROVED** [review]

**Pull request merged**